src/main/java/com/revature/model/Job.java

```diff
@@ -24,46 +24,40 @@
  */
 @Entity
 @Table(name = "jobs")
+
 public class Job {
-	

```

Comment:
However, the blank lines here serve to add readability, so they should remain.

Reply:
Changes are made as per suggestion